src/workerd/jsg/dom-exception-test.c++

```diff
@@ -11,7 +11,7 @@ namespace {
 
 V8System v8System;
 
-struct DOMExceptionContext: public Object {
+struct DOMExceptionContext: public Object, public ContextGlobal {
```

Comment:
Non-blocking nit: for these *test.c++ files, it would be nice to have a utility macro that can reduce boilerplate here... something like:

```
JSG_DECL_TEST_CONTEXT(DOMExceptionContext) {
  // ...
}
```

Such that it expands out to the `struct DOMExceptionContext: public Object, public ContextGlobal {`